src/python/pants/backend/core/tasks/prepare_resources.py

```diff
@@ -25,6 +104,11 @@ def register_options(cls, register):
     super(PrepareResources, cls).register_options(register)
     register('--confs', advanced=True, type=Options.list, default=['default'],
              help='Prepare resources for these Ivy confs.')
+    register('--use-jar', advanced=True, action='store_true', default=False,
+             help='Zip resources into a jar.')
```

Comment:
Please add a blurb about why this is useful, and why you might need to disable it (tests need to be written to assume jars.)